docs/reference/glossary.asciidoc

```diff
@@ -54,19 +54,26 @@
   pairs. The value can be a simple (scalar) value (eg a string, integer,
   date), or a nested structure like an array or an object. A field is
   similar to a column in a table in a relational database.
-  +
```

Comment:
I'd prefer to keep these lines in (also please revert the following deletes). They prevent paragraph breaks but make the source file more readable.

Reply:
Sorry, please I don't understand your point her

Reply:
The point was that the added free line in the documentation source is more readable as it separates the following paragraph visually, while the "+" prevents this from happening in the final page. I think this was intentional and shouldn't be changed as part of this PR, so please revert these changes.